lib/cErrors.mli

```diff
@@ -34,7 +34,7 @@ exception UserError of Pp.t
 (** Main error signaling exception. It carries a header plus a pretty printing
     doc *)
 
-val user_err : ?loc:Loc.t -> ?info:Exninfo.info -> Pp.t -> 'a
+val user_err : ?loc:Loc.t -> ?info:Exninfo.info -> ?qf:Quickfix.t list -> Pp.t -> 'a
```

Comment:
What happens to the quickfix data? It doesn't seem like anything reads it.

Reply:
done